packages/benchmarks/lit-element/class-map/index.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
```

Comment:
Note: This license date is from when this code was initially written. I did modify and simplify it, and maybe it makes sense to set this date to the commit date (2022).